### DISCUSSION TITLE
Grammarly估值衝破 130 億美元

### DISCUSSION THREAD

**Discussion by zixunn** [1101-GS4719-C/Python-Community-GS4719-C#263] [General]
Grammarly是個有各種工具集的寫作工具，像是平常寫郵件時若有文法問題就會即時通知我們並提供正確拼法，對我來說它就是一個隱藏在電腦背後的寫作小幫手，沒想到一週前Grammarly宣布完成2億美元新一輪融資，最新估值達130億美元，加入全球百億獨角獸俱樂部，成為美國估值最高獨角獸第十名。
[幫忙改拼字文法的「小軟體」估值衝破 130 億美元](https://technews.tw/2021/12/05/grammarly-valuation/)

**Comment by chun1203** [comment]
之前學測前改作文也有用過，相當的方便！

**Comment by www0000596** [comment]
真的！能幫忙注意到小細節，在後續審閱時候也減輕了一些負擔🙌🏻
在這一部分的確滿足了很多人的需求

**Comment by 107602505** [comment]
一直都有在Youtube上看過很多次Grammarly的廣告，原來它也是利用AI辨識，都不知道它是如此強大的寫作工具，除了簡單的文法、標點符號檢查，甚至還有抄襲檢查和語氣和語調寫作建議，真的太厲害了!

**Comment by 109302003** [comment]
第一次知道這個工具，感謝分享！

**Comment by JAIMEHUNG** [comment]
原來這也有用到AI!，這讓我想到GOOGLE 文件也有抓錯字的功能，但不知道是否也有使用到AI。

**Comment by ghost** [comment]
原來每次開yt都會看到的廣告已經做這麼大了嗎XD

**Comment by luo910227** [comment]
真的是每次開youtube都常常看到這廣告，但不知道原來它做這麼大而且有應用AI技術，挺令人讚嘆，而且還有這麼多功能!

**Comment by cody910120** [comment]
這酷東西真牛逼，以後可以常常用ㄟ！

**Comment by mesakiyuta** [comment]
英文老師有推薦這軟體，練寫作時可以用這個來自我檢查。
https://writeandimprove.com/
這網站也不錯，會幫你的文章評分，挑出不順的地方
